manager/assets/modext/widgets/security/modx.panel.groups.roles.js

```diff
@@ -69,6 +80,7 @@ Ext.extend(MODx.panel.GroupsRoles,MODx.FormPanel,{
                             ,useSplitTips: true
                             ,monitorResize: true
                             ,width: 270
+                            ,minWidth: 270
                             ,minSize: 270
                             ,maxSize: 400
```

Comment:
Maybe yes, before it didn't work at all so I saw it only when tried to test this changes.